docs/ai-agent/ai-agent-spans.md

```diff
@@ -0,0 +1,185 @@
+| `ai_agent.workflow.config`     | object | Configuration settings for the workflow.                  | `{"setting1": "value1"}`          | Recommended       | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+| `ai_agent.workflow.start_time` | string | Timestamp when the workflow started.                      | `2024-12-06T10:00:00Z`            | Required          | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+| `ai_agent.workflow.end_time`   | string | Timestamp when the workflow ended.                        | `2024-12-06T11:00:00Z`            | Required          | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+| `ai_agent.workflow.end_state`  | string | Final state of the workflow (e.g., `success`, `failure`). | `success`                         | Required          | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+| `ai_agent.workflow.end_reason` | string | Reason for the workflow's end, if applicable.             | `Completed all tasks`             | Recommended       | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+| `ai_agent.workflow.system`     | string | System or environment where the workflow is executed.     | `CrewAI`, `LangGraph`, `AutoGen`  | Recommended       | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+
+### Agent Attributes
+
+| Attribute                      | Type   | Description                                | Example                          | Requirement Level | Stability    |
+| ------------------------------ | ------ | ------------------------------------------ | -------------------------------- | ----------------- | --- |
+| `ai_agent.agent.name`          | string | Name of the agent.                         | `Researcher Bot`                 | Required          | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+| `ai_agent.agent.role`          | string | Role assigned to the agent.                | `Data Collector`                 | Required          | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+| `ai_agent.agent.backstory`     | string | Background story or context for the agent. | `Specializes in web data mining` | Required          | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+| `ai_agent.agent.workflow_name` | string | Name of the workflow the agent is part of. | `Data Processing Pipeline`       | Required          | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
+| `ai_agent.agent.model`         | string | Underlying model powering the agent.       | `gpt-4`                          | Required          | ![Experimental](https://img.shields.io/badge/-experimental-blue)    |
```

Comment:
how's agent model is different from `gen_ai.request.model`?

Reply:
+1